modules/Campaigns/WizardHome.html

```diff
@@ -37,59 +37,75 @@
+    <input type="hidden" id="action" name="action" = "WizardHome">
+    <input type="hidden" id="process_form" name="process_form" value='false'>
+    <input type="hidden" id="return_module" name="return_module" value="{$RETURN_MODULE}">
+    <input type="hidden" id="return_id" name="return_id" value="{$RETURN_ID}">
+    <input type="hidden" id="return_action" name="return_action" value="{$RETURN_ACTION}">
+    <input type="hidden" id="record" name="record" value="{$ID}">
+    <input type="hidden" id="direct_step" name="direct_step" value="1">
+    <input type="hidden" id="campaign_id" name="campaign_id" value="">
+    <input type="hidden" id="wiz_mass" name="wiz_mass" value="">
+    <input type="hidden" id="mode" name="mode" value="">
+
 
+    <table class='other view' cellspacing="1">
+        <tr>
+            <td rowspan='2' width='10%' scope='row' style='vertical-align: top;'>
+                <div id='nav'>
+                    {$NAV_ITEMS}
 
+                </div>
 
-<table class='other view' cellspacing="1">
-<tr>
-<td rowspan='2' width='10%' scope='row' style='vertical-align: top;'>
-<div id='nav' >
-{$NAV_ITEMS}
+            </td>
 
-</div>
+            <td class='edit view' rowspan='2' width='100%'>
+                {$CAMPAIGN_DIAGNOSTIC_LINK}
+                <p>
+                    <table width="100%" border="0" cellspacing="1" cellpadding="0">
+                        <tr>
+                            <td>
+                                <div id='campaign_summary'>{$CAMPAIGN_TBL}</div>
+                <p>
+            </td>
+        </tr>
+        <tr>
+            <td>&nbsp;</td>
+        </tr>
+        <tr>
+            <td>
+                <p>
+                <div id='campaign_trackers'>{$TRACKERS_TBL}</div>
+            </td>
+        </tr>
+        <tr>
+            <td>&nbsp;</td>
+        </tr>
+        <tr>
+            <td>
+                <p>
+                <div id='campaign_targets'>{$TARGETS_TBL}</div>
+            </td>
+        </tr>
+        <tr>
+            <td>&nbsp;</td>
+        </tr>
+        <tr>
+            <td>
+                <p>
+                <div id='campaign_marketing'>{$MARKETING_TBL}</div>
+            </td>
+        </tr>
+    </table>
 
-</td>
 
-<td class='edit view' rowspan='2' width='100%'>
-{$CAMPAIGN_DIAGNOSTIC_LINK}<p>
-<table  width="100%" border="0" cellspacing="1" cellpadding="0" >
-	<tr><td>
-		<div id='campaign_summary'>{$CAMPAIGN_TBL}</div><p>
-	</td></tr>
-	<tr><td>&nbsp;</td></tr>
-	<tr><td>
-		<p><div id='campaign_trackers'>{$TRACKERS_TBL}</div>
-	</td></tr>
-	<tr><td>&nbsp;</td></tr>			
-	<tr><td>
-		<p><div id='campaign_targets'>{$TARGETS_TBL}</div>
-	</td></tr>
-	<tr><td>&nbsp;</td></tr>
-	<tr><td>
-		<p><div id='campaign_marketing'>{$MARKETING_TBL}</div>
-	</td></tr>
-</table>
-		
-</td></tr></table>
-<script>
-{$MSG_SCRIPT}
-</script>
+    </td></tr></table>
+    <script>
+      {
+        $MSG_SCRIPT
+      }
```

Comment:
Hi @Dillon-Brown, maybe here we should keep `{$MSG_SCRIPT}` as its part of the template engine syntax. Though keeping the correct indentation. What do you think?

Reply:
Pushed!